doc/fh3/app/create.md

```diff
@@ -2,23 +2,30 @@ fhc-create(1)
 =============
 ## SYNOPSIS
 
- fhc app create --project=<project> --title=<title> [--template=<template>] [--repo=<repo>] [--branch=<branch>]
+ fhc app create --project=<project> --title=<title> --type=cloud_nodejs [--template=<template>] [--repo=<repo>] [--branch=<refs/heads/my-branch-name>]
 
 ## EXAMPLES
 
-  fhc app create --project=1a2b3c --title=My New App --type=cloud_nodejs                                Creates a new hybrid app from template
-  fhc app create --project=1a2b3c --title=My New App --repo=git:///some.com/repo.git --branch=master    Creates a new hybrid app from a git repo
+  fhc app create --project=1a2b3c --title=My New App --type=cloud_nodejs
 
+  * Creates a new hybrid app from template.
+
+
+  fhc app create --project=1a2b3c --title=My New App --type=cloud_nodejs --repo=git:///some.com/repo.git --branch=master    
```

Comment:
Or, can you test it without `refs/head/...`? If it works only for `master` then we should document it. Otherwise always default to the format with `refs/head/...`